README.md

```diff
@@ -283,7 +282,7 @@ then,
 Other useful commands include:
 
 ```bash
-poetry run airflow resetdb # In case the database connection is set up incorrectly
+poetry run airflow db reset # In case the database connection is set up incorrectly
```

Comment:
Here as well, let's make sure it's compatible between the two major versions.

Reply:
I reverted the changes to commands. The rest of the edits were independent of which Airflow version is used, so now everything should be fine.